packages/map-balloon/src/MapBalloon.js

```diff
@@ -3,6 +3,8 @@ import PropTypes from "prop-types";
 import values from "object.values";
 import Text, { fontTypes } from "@crave/farmblocks-text";
 import { colors, fontSizes } from "@crave/farmblocks-theme";
+import { Flex, Box } from "grid-styled";
```

Comment:
What advantages bringing in a grid layout framework will give us? Craity / readabality? Compatibility? Future-proofness? It looks like the demands for a layout system inside a balloon currently is not much today, is the trade-of in styling declaratively with components vs using plain css good for us? Is the time for this right? I dont have the answer, just curious...:)

Reply:
Yeah, it's not really necessary.